components/coda/coda.app.mjs

```diff
@@ -1,11 +1,158 @@
+import axios from "axios";
+
 export default {
   type: "app",
   app: "coda",
-  propDefinitions: {},
+  propDefinitions: {
+    title: {
+      type: "string",
+      label: "Doc Title",
+      description: "Title of the doc",
+      optional: true,
+    },
+    folderId: {
+      type: "string",
```

Comment:
Is there an API endpoint for listing folders so that we can present this via async options, as well?

Reply:
Coda API doesn't have an endpoint for listing folders, as seen in the folderId description in the following link.

> The ID of the folder within which to create this doc. Defaults to your "My docs" folder in the oldest workspace you joined; this is subject to change. You can get this ID by opening the folder in the docs list on your computer and grabbing the folderId query parameter.

https://coda.io/developers/apis/v1#operation/createDoc

Reply:
This is the same for workspaceIds

Reply:
Ack, thanks